.github/workflows/changesets.yml

```diff
@@ -21,19 +18,30 @@ jobs:
               with:
                   fetch-depth: 0
 
+            - name: setup pnpm
+              uses: pnpm/action-setup@v4
+
             - name: setup node.js
               uses: actions/setup-node@v3
```

Comment:
could you bump `checkout` and `setup-node` to v4 as long as we're updating this?